archinstall/lib/unicode.py

```diff
@@ -0,0 +1,40 @@
+import unicodedata
```

Comment:
@correctmost would it make sense to put this into the `utils.py` rather to avoid too many modules with utility functions?

Reply:
yeah lets move it in there that will do